src/whitenoise/event_tracker.rs

```diff
@@ -174,15 +183,24 @@ impl EventTracker for WhitenoiseEventTracker {
         &self,
         event_id: &EventId,
         pubkey: &PublicKey,
+        event_created_at: Option<DateTime<Utc>>,
+        event_kind: Option<u16>,
```

Comment:
Maybe it'd be more natural to just receive the event in this method and internally extract all we need?